aptos-move/aptos-aggregator/src/delta_change_set.rs

```diff
@@ -327,18 +343,22 @@ impl DeltaChangeSet {
 
     /// Converts deltas to a vector of write ops. In case conversion to a write op
     /// failed, the error is propagated to the caller.
-    pub(crate) fn take_materialized(
+    pub fn take_materialized(
```

Comment:
nit: typically a function that consumes self is called into/try_into_xxx, maybe this should be try_into_write_ops or something similar